cleanup_switzerland/models/database_cleanup.py

```diff
@@ -52,7 +51,10 @@ def clean_mail_message(self):
         self.env.cr.execute(
             "DELETE FROM mail_message WHERE model IS NULL"
         )
-        models = self.env['ir.model'].search([]).mapped('model')
+
+        self.env.cr.execute(sql.SQL("SELECT DISTINCT model FROM mail_message"))
+        models = [m[0] for m in self.env.cr.fetchall()]
+
```

Comment:
The cron was looping on every existing models. Many of them were raising errors because they don't have relations in database (probably the abstract models, almost same behaviour in Odoo10).
So I edited to loop only over models referenced in the mail_message relation.

With an empty database it is really faster but I wonder if it is a efficient with the prod database